src/util/dom_style.ts

```diff
@@ -1,20 +1,15 @@
-//@ts-nocheck
 // TODO this file needs to go away, cross browser style support is not fabricjs domain.
-
 /**
  * wrapper for setting element's style
  * @param {HTMLElement} element
  * @param {Object | string} styles
  */
-export function setStyle(element, styles) {
+export function setStyle(element: HTMLElement, styles: Record<string, string>) {
   const elementStyle = element.style;
   if (!elementStyle) {
     return;
-  } else if (typeof styles === 'string') {
-    element.style.cssText += ';' + styles;
```

Comment:
I remember a bug that was solved with this....
I might be wrong.
It was to do with restoring the lower canvas style.

Reply:
ok... let me try to look at the history and see if i figure out

Reply:
i ll revert the changes to this function, this deserve a mini PR and an extra mini clarificaiton.